src/analysisd/format/to_json.c

```diff
@@ -153,6 +153,83 @@ char* Eventinfo_to_jsonstr(const Eventinfo* lf, bool force_full_log)
                 cJSON_Delete(mitre_technique_array);
             }
         }
+        if(lf->generated_rule->mitre_technique_id && lf->generated_rule->mitre_tactic_id) {
+            cJSON * mitre = NULL;
+            cJSON * element = NULL;
+            int tactic_array_size;
+            technique_data * data_technique = NULL;
+
+            cJSON_AddItemToObject(rule, "mitre", mitre = cJSON_CreateObject());
```

Comment:
What if a rule has both deprecated and new configuration tags? Will you create `mitre` object twice?